bungee/src/main/java/co/aikar/commands/BungeeCommandManager.java

```diff
@@ -28,13 +28,11 @@
 import net.md_5.bungee.api.CommandSender;
 import net.md_5.bungee.api.plugin.Command;
 import net.md_5.bungee.api.plugin.Plugin;
+import net.md_5.bungee.api.plugin.PluginManager;
 
 import java.lang.reflect.Method;
 import java.lang.reflect.Parameter;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Locale;
-import java.util.Map;
+import java.util.*;
```

Comment:
star import ;)

Reply:
IDE must've done that. I'll fix it in a sec